### ISSUE TITLE
Feature request: Move window without resizing

### ISSUE THREAD

**Issue by marlonrichert** [eczarny/spectacle#884]
I often find that I want to center a window in the left half or right half of the screen, but without resizing it.

**Comment by quazar0** [comment]
I would also like this.  Most of the time I would like to move the window to the left or right side of the screen, without resizing, because I am frequently editing code, and my editor window is already set to show the source code at the correct width.  And if it resizes, then it grows larger and wastes screen space that is not being used, or it shrinks the window size and then I cannot see some of my code.
This may be as easy as providing a single on/off switch that says not to resize on move.
BTW, thanks for the good work; it is appreciated.

**Comment by rxhanson** [comment]
I recently rewrote Spectacle in Swift as a new app called [Rectangle](https://rectangleapp.com), and this is one of the items that I added in. 

Note that Rectangle's move left/right/up/down actions will snap the window to the edge of the screen, centered, as opposed to centering the window in the screen half. In my testing, this felt more natural, especially on screen sizes that weren't really large. It also simplifies the logic quite a bit.

Feel free to try it out.
